cashctrl_api/client.py

```diff
@@ -256,6 +288,10 @@ def update_categories(self, resource: str, target: List[str],
                     params = {'name': nodes[i]}
                     if parent_path:
                         params['parentId'] = categories[parent_path]
+                    elif resource == 'account':
+                        params['parentId'] = categories.get(parent_path, 1)
```

Comment:
That line seems odd. 
- Are we attaching all account categories without existing path simply to 'Assets' (the first category)?
- Better raise an error.